.github/workflows/linux_build_test.yml

```
@@ -49,32 +46,37 @@ jobs:
         ]
 
     container:
-      image: ghcr.io/svalinn/dagmc-ci-ubuntu-${{ matrix.ubuntu_versions }}-${{ matrix.compiler }}-ext-hdf5_${{ matrix.hdf5_versions }}-moab_${{ matrix.moab_versions }}:stable
+      image: ghcr.io/${{ github.repository_owner }}/dagmc-ci-ubuntu-${{ matrix.ubuntu_versions }}-${{ matrix.compiler}}-ext-hdf5_${{ matrix.hdf5_versions}}-moab_${{ matrix.moab_versions }}/moab:latest
 
     steps:
       - name: Checkout repository
         uses: actions/checkout@v3
         with:
           submodules: recursive
 
-      - name: Setup
-        run: |
-          echo "MOAB_VERSION=${{ matrix.moab_versions }}" >> $GITHUB_ENV
-          echo "COMPILER=${{ matrix.compiler }}" >> $GITHUB_ENV
-          echo "HDF5_VERSION=${{ matrix.hdf5_versions }}" >> $GITHUB_ENV
-          echo "REPO_SLUG=${GITHUB_REPOSITORY}" >> $GITHUB_ENV
-          echo "PULL_REQUEST=$(echo  $GITHUB_REF | cut -d"/" -f3)" >> $GITHUB_ENV
-          echo "DOUBLE_DOWN="OFF"" >> $GITHUB_ENV
-          echo "PYTHONPATH=/root/build_dir/moab/bld/pymoab/lib/python3.8/site-packages:${PYTHONPATH}" >> $GITHUB_ENV
-          ln -s $GITHUB_WORKSPACE /root/build_dir/DAGMC
-
       - name: Building DAGMC
         run: |
-          cd $GITHUB_WORKSPACE
-          git config --global --add safe.directory /__w/DAGMC/DAGMC
-          CI/scripts/install.sh
+          ln -s $GITHUB_WORKSPACE /root/build_dir/DAGMC
+          mkdir -p ./build
+          cd ./build
+          git config --global --add safe.directory $GITHUB_WORKSPACE
```

```suggestion
          git config --global --add safe.directory /root/build_dir/DAGMC
```
but we would have to put the path here

---

I think this stays no matter what - it's `GITHUB_WORKSPACE` we need to declare safe